sdk/src/utils/xmp_inmemory_utils.rs

```diff
@@ -261,4 +317,54 @@ mod tests {
         println!("{xmp}");
         assert_eq!(unicorn, Some(PROVENANCE.to_string()));
     }
+
+    #[cfg_attr(not(target_arch = "wasm32"), actix::test)]
+    #[cfg_attr(
+        all(target_arch = "wasm32", not(target_os = "wasi")),
+        wasm_bindgen_test
+    )]
+    #[cfg_attr(target_os = "wasi", wstd::test)]
+    async fn test_broken_xmp_read_write_stream() {
+        let source_bytes = include_bytes!("../../tests/fixtures/broken.jpg");
```

Comment:
Should we rename the file?

Reply:
You can ask Jen if we can use it.

Reply:
We try to keep our repo images small, this one is over 6mb.